samples/utils/CommandLineUtils.cpp

```diff
@@ -677,6 +691,9 @@ namespace Utils
         {
             returnData.input_port = atoi(cmdUtils.GetCommandOrDefault(m_cmd_port_override, "0").c_str());
         }
+        returnData.input_accessKeyId = cmdUtils.GetCommandOrDefault(m_cmd_access_key_id, "");
+        returnData.input_secretAccessKey = cmdUtils.GetCommandOrDefault(m_cmd_secret_access_key, "");
+        returnData.input_sessionToken = cmdUtils.GetCommandOrDefault(m_cmd_session_token, "");
```

Comment:
Debatable: These options should be required when connecting with static creds. For that, a designated function is needed, which might be overkill. That's why debatable.